avogadro/avogadro.cpp

```diff
@@ -71,7 +71,7 @@ int main(int argc, char* argv[])
 
   // Set up the default format for our GL contexts.
   QSurfaceFormat defaultFormat = QSurfaceFormat::defaultFormat();
```

Comment:
In this case, the whole section can be commented out - as you currently have it, you access the default format to then set it as the default format. Better to comment out line 73 and 76 as well.